src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <SPI.h>        
+#include <NativeEthernet.h>
+#include <NativeEthernetUdp.h>
+
+constexpr int SUCCESSFUL = 1;
+constexpr int FAILIURE = 0;
+
+byte macAddressTeensy[] = { 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED };
+IPAddress ipAddressTeensy(10, 0, 0, 1);
+IPAddress switchAddress(129, 241, 187, 1);
+IPAddress targetAddress(10, 0, 0, 2);
+
+unsigned int localPort = 8888;
+unsigned int targetPort = 8888;
+
+char stringSent[] = "No packets recieved, 5 second cooldown set";
+char recieveConfirmed[] = "PacketRecieved";
+char recievedString[UDP_TX_PACKET_MAX_SIZE];
+
+// An EthernetUDP instance to let us send and receive packets over UDP
+
+EthernetUDP Udp;
```

Comment:
Extract to separate .hpp/.cpp file if possible.

Reply:
In PlatformIO, nothing can be _defined_ for some reason. Therefore, constants definitions were moved to source file.